src/aggregator/integration/placement_change_test.go

```diff
@@ -237,45 +222,42 @@ func TestPlacementChange(t *testing.T) {
 		clock.SetNow(data.timestamp)
 
 		for _, mm := range data.metricWithMetadatas {
-			idx := getServerIndex(mm.metric.ID(), initialPlacement)
-			require.NoError(t, clients[idx].writeTimedMetricWithMetadata(mm.metric.timed, mm.metadata.timedMetadata))
-		}
-		for _, c := range clients {
-			require.NoError(t, c.flush())
+			require.NoError(t, client.writeTimedMetricWithMetadata(mm.metric.timed, mm.metadata.timedMetadata))
```

Comment:
Trying to understand the change here:
I see that previously the write was done to the server which was determined by metric ID. But now it always writes to server 0. Is this really OK?

Reply:
Before, when we used bespoke client for tests, each instance of the client would maintain a TCP connection to a single server. Thus we had to choose appropriately which client to use to reach target server. Nowadays, we use a proper aggregator client which does the routing itself, so only one client instance is enough.

Reply:
So this client that gets created off the `servers[0]` (`client := servers[0].newClient(t)`) can write to all servers? 
This looks confusing. Maybe possible to refactor creation for the client from the actual server if it's unrelated? At worst, consider adding a comment explaining this.

Reply:
Added this convenience method with a comment: https://github.com/m3db/m3/pull/3744/commits/edcf4f57970c03213b61186b9d21eef8cbc15220#diff-49c6046db805a84ae1fc5baf10f6fef5ebfc98335a0573a6a2f38d569912e829R463-R469